deploy/resources/v0.8.0/addons/pipelinetemplates/s2i-go/s2i-go-build-deploy.yaml

```diff
@@ -0,0 +1,44 @@
+---
```

Comment:
@pradeepitm12 why is this added to `0.8.0` ?

Reply:
+1
@pradeepitm12 could you move this to `deploy/resources/v0.9.2/addons`